var/spack/repos/builtin/packages/prmon/package.py

```diff
@@ -11,14 +11,15 @@ class Prmon(CMakePackage):
     """Standalone monitor for process resource consumption."""
 
     homepage = "https://github.com/HSF/prmon/"
-    url      = "https://github.com/HSF/prmon/archive/v1.1.1.tar.gz"
+    url      = "https://github.com/HSF/prmon/archive/refs/tags/v2.2.0.zip"
     git      = "https://github.com/HSF/prmon.git"
 
     maintainers = ['vvolkl']
 
     version("master", branch="master")
-    version('2.1.1', sha256='b3d9327717caa857dfda5bee2308375c0c000817275160ed09970d805a1ec9d6')
-    version('1.1.1', sha256='5f074b05af2a12e2726c33f6a6e9e8e59ee0c4fb5fe056deb38abacd1bb6bf03')
+    version('2.2.0', sha256='7b3b887c679279f0e666e8c8c58ca1a22a328b8b94ecff09e61795a6a83e8ce5')
+    version('2.1.1', sha256='302d7a3fc5a403143d794e16dca1949e3d13e46cf9857dfaad4e374f4a468df2')
+    version('1.1.1', sha256='a6e6486bbc4d6cddf73affe07d9ff7948a424c9a02b3cdd5bbe5c6cafa06af2e')
```

Comment:
Confirmed these are valid sha256 for the new url/extension.

Reply:
@vvolkl FWIW. You didn't need to switch to `zip` to deal with the `sha256` issue.  You only needed to change the `sha256` values to match what spack uses since it always performs the equivalent of `spack checksum <package>@<version>`.

Reply:
Yes, but I would have forgotten all about the issue by the time of the next release, and since I always do `spack checksum <package>` we'd be sure to have the same conversation again ...

Reply:
Fair enough.